creator/projects/schema.py

```diff
@@ -132,7 +124,16 @@ def mutate(self, info, input):
             raise GraphQLError(
                 f"Study already has a {input['workflow_type']} project."
             )
-        project = create_project(study, "HAR", input["workflow_type"])
+
+        regex = re.compile("[~`!@#$%^&*()+}{:;<>?/\\\\|]")
```

Comment:
I think we may need to also blacklist `_` because it's not valid for urls.

Reply:
Updated